int32 encoder_count
int16 input_voltage
int16 motor_velocity
